apis/v1beta1/shared_types.go

```diff
@@ -295,6 +317,11 @@ const (
 	// match any Listeners in the Gateway.
 	RouteReasonNoMatchingParent RouteConditionReason = "NoMatchingParent"
 
+	// This reason is used with the "Accepted" condition when there are
+	// multiple ParentRefs that are not distinct. See the ParentRef
+	// definition for the definition of distinct.
+	RouteReasonParentsNotDistinct RouteConditionReason = "ParentsNotDistinct"
+
 	// This reason is used with the "Accepted" condition when a value for an Enum
 	// is not recognized.
 	RouteReasonUnsupportedValue RouteConditionReason = "UnsupportedValue"
```

Comment:
I'm pretty happy with the language changes in here :+1: 

We should make sure we hold for @robscott before merging.

/hold